examples/cpem-add-on/versions.tf

```diff
@@ -1,4 +1,9 @@
 terraform {
+  provider_meta "equinix" {
+    # TEMPLATE: Set the name of the module below 
+    module_name = "template"
```

Comment:
I'm not sure that we want/need to set the `module_name` for the example modules.  If we did, I think we'd want it to be something like `<root_module_name>/<example_module_name>`; in this case, that would be `equinix-kubernetes-cluster/cpem-add-on`.

Reply:
I like `equinix-kubernetes-cluster/cpem-add-on`. I noticed in k8s-addons that we used a space as the separator.

https://github.com/equinix-labs/terraform-equinix-kubernetes-addons/blob/b5b6881eb33dbcb90e8e8c79ba7147ae3c1e983a/modules/metallb/versions.tf#LL16C1-L16C1